datadog_checks_dev/datadog_checks/dev/tooling/commands/release.py

```diff
@@ -522,32 +522,41 @@ def tag(check, version, push, dry_run):
     else:
         checks = [check]
 
+    # Check for any new tags
+    tagged = False
+
     for check in checks:
-        echo_success('Check `{}`'.format(check))
+        echo_info('{}:'.format(check))
 
         # get the current version
         if not version:
             version = get_version_string(check)
 
         # get the tag name
         release_tag = get_release_tag_string(check, version)
-        echo_waiting('Tagging HEAD with {}'.format(release_tag))
+        echo_waiting('Tagging HEAD with {}... '.format(release_tag), indent=True, nl=False)
 
         if dry_run:
             version = None
+            click.echo()
             continue
 
         result = git_tag(release_tag, push)
 
-        # For automation we may want to cause failures for extant tags
         if result.code == 128 or 'already exists' in result.stderr:
-            echo_warning('Tag `{}` already exists, skipping...'.format(release_tag))
+            echo_warning('already exists')
```

Comment:
why not put the line from 557 here?

Reply:
This is in a loop

Reply:
Won't tagged be overwritten a few times, then?

Reply:
No b/c that would require `--force` which we don't do

https://git-scm.com/docs/git-tag
https://github.com/DataDog/integrations-core/blob/153d822d2478a6cc4e49df0de0a7bc7bc25d885f/datadog_checks_dev/datadog_checks/dev/tooling/git.py#L92

Reply:
sorry, I meant the boolean variable tagged

Reply:
Ah, then yes. Which is fine as it's only every updated to True

Reply:
okay, and that's fine?

Reply:
Oh I suppose I see

Reply:
A tag can exist without it being bad that it exists